app/dashboard/views.py

```diff
@@ -1518,7 +1519,21 @@ def change_bounty(request, bounty_id):
         for key in keys:
             value = params.get(key, '')
             old_value = getattr(bounty, key)
+
+            if key == 'bounty_reserved_for_user' and value is not None:
+                if old_value is not None:
+                    old_value = old_value.id
+                else:
+                    old_value = None
```

Comment:
If `old_value` is `None` as validated by the `if` above, there's no need to reassign this value.